impeller/compiler/compiler.cc

```diff
@@ -139,7 +139,7 @@ Compiler::Compiler(const fml::Mapping& source_mapping,
       break;
     case TargetPlatform::kFlutterSPIRV:
       spirv_options.SetOptimizationLevel(
-          shaderc_optimization_level::shaderc_optimization_level_size);
+          shaderc_optimization_level::shaderc_optimization_level_zero);
```

Comment:
Curious, why?

Reply:
The transpiler is missing some ops: https://github.com/flutter/flutter/issues/105396. Added a comment.

Also it's the level being used by the current test harness: https://github.com/flutter/engine/blob/d0ec2205b34e89f4f8c887d1d378abad91ed9df9/lib/spirv/test/glsl_to_spirv.cc#L39